tron/actioncommand.py

```diff
@@ -3,6 +3,7 @@
 
 import logging
 import os
+from pipes import quote
```

Comment:
didn't know about this! While looking this up, I saw that it's deprecated after python2.7.

You might want to import this with six.moves.shlex_quote to help maksym out :)